libs/react-ui/src/lib/main-navigation/main-navigation.scss

```diff
@@ -64,7 +65,7 @@
 
   &__toggle {
     &[aria-expanded=true] {
-      font-weight: $veera-font-weight-delta;
+      font-weight: 700;
```

Comment:
Why constants instead of variables?

Reply:
These variables does not exist in styles. Requires changing general styles. A separate ticket maybe?

Reply:
Ok for now.